pkg/util/github/file.go

```diff
@@ -21,3 +27,30 @@ func (c *Client) CreateFile(content []byte, filePath, targetBranch string) error
 	_, _, err := c.Repositories.CreateFile(c.Context, owner, c.Repo, filePath, opt)
 	return err
 }
+
+func (c *Client) PushLocalPath(repoPath, branch string) error {
+	if err := filepath.Walk(repoPath, func(path string, info fs.FileInfo, err error) error {
```

Comment:
`WalkDir` is better than `Walk`, but we are not a performance-sensitive program and can ignore it for now.

https://pkg.go.dev/path/filepath#Walk